assets/js/wpuf-form-builder-components.js

```
@@ -186,11 +185,196 @@ Vue.component('builder-stage', {
+                Swal.fire({
+                    title: "Oops...",
+                    text: "You already have this field in the form"
+                });
+                return;
+            }
+
+            this.$store.commit('clone_form_field_element', payload);
+        },
+
+        delete_field: function(index) {
+            var self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)